afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt

```diff
@@ -51,14 +52,12 @@ class AfterpayPriceBreakdown(context: Context, attrs: AttributeSet?) : FrameLayo
         layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT)
     }
 
-    // The terms and conditions are currently strongly tied to the configuration currency however
-    // this may change in the future to enable accounts paying in a foreign currency
+    // The terms and conditions are tied to the configured locale on the configuration
     private val infoUrl: String
-        get() = when (Afterpay.configuration?.currency?.currencyCode) {
-            "AUD" -> "https://static-us.afterpay.com/javascript/modal/au_rebrand_modal.html"
-            "NZD" -> "https://static-us.afterpay.com/javascript/modal/nz_rebrand_modal.html"
-            "CAD" -> "https://static-us.afterpay.com/javascript/modal/ca_rebrand_modal.html"
-            else -> "https://static-us.afterpay.com/javascript/modal/us_rebrand_modal.html"
+        get() {
+            val locale = Afterpay.configuration?.locale ?: Locale.US
+            val country = locale.country.toLowerCase(Locale.ROOT)
+            return "https://static-us.afterpay.com/javascript/modal/${country}_rebrand_modal.html"
```

Comment:
Very nice 👌

Reply:
Now you've reminded me that I should restrict the valid locale set like I have done on iOS

Reply:
Ah good idea. I've also noticed that `Afterpay.configuration?.locale ?: Locale.US` is performed in multiple places so maybe it's worth consolidating it?

Reply:
I thought this. I think it'll be something to address at the end once I've done currency formatting. The correct place to put it hopefully should be more apparent when everything is complete

Reply:
One thing you can do is create an extension on the nullable `Locale`, like is done with the `orEmpty()` extension on a nullable `List`:

```kotlin
internal fun Locale?.orDefault(): Locale = this ?: Locale.US
```